docs/development/ide-configuration.md

```diff
@@ -5,10 +5,31 @@ To fully benefit the ESLint and Jest testing frameworks, you must properly set u
 This document describes how to directly integrate in your IDE some tools the npm build is relying on. For the IDE not
 mentioned here, the following should help you to know which extensions or plugins to add to your development environment. 
 
+* [Gitpod](#Gitpod) 
 * [Visual Studio Code](#visual-studio-code)
 * [IntelliJ](#intellij)
 
 
+### Gitpod
+
+It is advised to read the information written below.
+If you are however familiar with Gitpod you can jump there and start coding right away by one click: [![Open in Gitpod](https://gitpod.io/button/open-in-gitpod.svg)](https://gitpod.io/#https://github.com/process-analytics/bpmn-visualization-js)
+
+#### Git integration
+
+The best way to learn is to read oficial [Gitpod doc git](https://www.gitpod.io/docs/git/)  
+
+You can easily create Pull Requests directly from Gitpod [Gitpod doc pull requests](https://www.gitpod.io/docs/pull-requests/)  
+
+Once you have logged in to Gitpod do not forget to give it access to write in public repositories:
+
+1. Visit: [https://gitpod.io/access-control/](https://gitpod.io/access-control/)
+2. Check the box titled: write public repos
+3. Click on Update button
+
+After that quick setup you are able to contribute
```

Comment:
:question: should we document that once started, the workspace is ready to build and run the dev server and tests?

Reply:
good idea 👍